src/ert/parsing/config_keywords.py

```diff
@@ -1,786 +1,125 @@
-
-
-def init_user_config() -> Dict[str, SchemaItem]:
-    schema = {}
-    for item in [
-        workflow_job_directory_keyword(),
-        load_workflow_keyword(),
-        load_workflow_job_keyword(),
-        float_keyword(ENKF_ALPHA_KEY),
-        float_keyword(STD_CUTOFF_KEY),
-        update_setting_keyword(),
-        string_keyword(UPDATE_LOG_PATH_KEY),
-        string_keyword(MIN_REALIZATIONS_KEY),
-        int_keyword(MAX_RUNTIME_KEY),
-        string_keyword(ANALYSIS_SELECT_KEY),
-        stop_long_running_keyword(),
-        analysis_copy_keyword(),
-        analysis_set_var_keyword(),
-        string_keyword(ITER_CASE_KEY),
-        int_keyword(ITER_COUNT_KEY),
-        int_keyword(ITER_RETRY_COUNT_KEY),
-        # the two fault types are just added to the config object only to
-        # be able to print suitable messages before exiting.
-        gen_kw_keyword(),
-        schedule_prediction_file_keyword(),
-        string_keyword(GEN_KW_TAG_FORMAT_KEY),
-        gen_data_keyword(),
-        summary_keyword(),
-        surface_keyword(),
-        field_keyword(),
-        single_arg_keyword(ECLBASE_KEY),
-        existing_path_keyword(DATA_FILE_KEY),
-        existing_path_keyword(GRID_KEY),
-        path_keyword(REFCASE_KEY),
-        string_keyword(RANDOM_SEED_KEY),
-        num_realizations_keyword(),
-        run_template_keyword(),
-        path_keyword(RUNPATH_KEY),
-        path_keyword(DATA_ROOT_KEY),
-        path_keyword(ENSPATH_KEY),
-        single_arg_keyword(JOBNAME_KEY),
-        forward_model_keyword(),
-        simulation_job_keyword(),
-        data_kw_keyword(),
-        define_keyword(),
-        existing_path_keyword(OBS_CONFIG_KEY),
-        existing_path_keyword(TIME_MAP_KEY),
-        single_arg_keyword(GEN_KW_EXPORT_NAME_KEY),
-        history_source_keyword(),
-        path_keyword(RUNPATH_FILE_KEY),
-        int_keyword(MAX_SUBMIT_KEY),
-        int_keyword(NUM_CPU_KEY),
-        queue_system_keyword(False),
-        queue_option_keyword(),
-        job_script_keyword(),
-        load_workflow_job_keyword(),
-        set_env_keyword(),
-        update_path_keyword(),
-        path_keyword(LICENSE_PATH_KEY),
-        install_job_keyword(),
-        install_job_directory_keyword(),
-        hook_workflow_keyword(),
-        existing_path_keyword(CONFIG_DIRECTORY_KEY),
-    ]:
-        schema[item.kw] = item
-        if item.kw in ALIASES:
-            for name in ALIASES[item.kw]:
-                schema[name] = item
-    return schema
+class ConfigKeys(str, Enum):
+    # Note: The values of these must correspond to
+    # those used in the ert config, unless we make
+    # the keys match up and set logic checking
+    # config keys against strings (e.g., in check_required)
+    # to use only names
+    # These keys are used as options in KEY:VALUE statements
+    BASE_SURFACE_KEY = "BASE_SURFACE"
+    DEFINE_KEY = "DEFINE"
+    FORWARD_INIT_KEY = "FORWARD_INIT"
+    GENERAL_KEY = "GENERAL"
```

Comment:
I think we should remove the `_KEY` suffix while we are at it.

Reply:
Done